client/app/cdn/dedicated/manage/logs/cdn-dedicated-manage-logs.routes.js

```diff
@@ -0,0 +1,12 @@
+angular.module('App').config(($stateProvider) => {
+  $stateProvider.state('app.networks.cdn.dedicated.logs', {
+    url: '/logs',
+    views: {
+      cdnView: {
+        templateUrl: 'cdn/dedicated/manage/logs/cdn-dedicated-manage-logs.html',
+        controller: 'CdnLogsCtrl',
```

Comment:
Please use a `Component` instead :) here is an example: https://github.com/ovh-ux/ovh-manager-dedicated/blob/e4470a79acd07fbbac2f141a49a3498f8bc5d093/client/app/components/expiration/service-expiration-label.component.js

and the route would like like this:
```suggestion
         component: 'cndLogs',
```